ubuntudesign/www.ubuntu.com
# kubernetes redirect

## Done

Redirect for the kubernetes page

## QA

go to `/containers/kubernetes` and see that you are redirected to `/cloud/kubernetes`
